build.sbt

```diff
@@ -17,7 +17,7 @@ lazy val core = (project in file ("modules/core"))
   .settings(
     name := "arugula-core",
     libraryDependencies ++= Seq(
-      "io.lettuce" % "lettuce-core" % "6.0.0+",
```

Comment:
when I did a reload of sbt I found 6.0.0 didn't work. had to go up to 6.3.0.RELEASE for it to pass again. We can go back and revise this once we have broad test coverage across many versions of lettuce.